src/main/java/com/softserveinc/ita/vsaroz/pageobjects/RozetkaPage.java

```diff
@@ -1,20 +1,28 @@
 package com.softserveinc.ita.vsaroz.pageobjects;
 
 import static com.codeborne.selenide.Selenide.*;
+import static java.lang.String.format;
 
 public class RozetkaPage {
 
-    private final String categoryTypeSelector = ("//div[@class='menu-wrapper menu-wrapper_state_static ng-star-inserted']//li[1]");
-    private final String laptopFilterCheckboxElement = ("//a[@title='Ноутбуки']");
-    private final String filteredCheckboxName = ("//label[@for='Dell']");
+    private final String categoryTypeSelector = "//div[@class='menu-wrapper menu-wrapper_state_static ng-star-inserted']//li[1]";
+    private final String laptopFilterCheckboxElement = "//a[@title='Ноутбуки']";
+    private final String filteredCheckboxName = "//label[@for='Dell']";
 
-    public void clickOnCategorySelector() { $x(categoryTypeSelector).click(); }
+    public void clickOnCategorySelector() {
+        $x(categoryTypeSelector).click();
+    }
 
-    public void clickOnLaptopFilter() { $x(laptopFilterCheckboxElement).click(); }
+    public void clickOnCategory() {
```

Comment:
rename, I think it should some operation on filter checkbox for ex: activateLaptopFilter